api/pkg/authz/enforcer/role.go

```diff
@@ -0,0 +1,6 @@
+package enforcer
+
+const (
+	MLPAdminRole      = "mlp.administrator"
+	ProjectReaderRole = "mlp.projects.reader"
```

Comment:
nit: why do we prefix the admin role variable with `MLP`, but we don't do this for the project reader role? Shall we make it consistent?

Reply:
Also, `ProjectsReaderRole` is reflecting this role's meaning better

Reply:
Yeah the naming could be confusing here. So we need two different types of role: the project readers role, which is able to read every single project in MLP. There's also another type of role, that can only read specific project. I am brain storming right now as to what's the best name to avoid confusion, especially if we are to define them via template string all in a single file.